ext/opcache/Optimizer/sccp.c

```diff
@@ -844,13 +844,17 @@ static inline int ct_eval_func_call(
 				|| zend_string_equals_literal(name, "urldecode")
 				|| zend_string_equals_literal(name, "rawurlencode")
 				|| zend_string_equals_literal(name, "rawurldecode")
-				|| zend_string_equals_literal(name, "php_uname")) {
```

Comment:
This should not be removed. I think it was added years ago because some web frameworks used it with a constant frequently, and as I mentioned, it's currently completely correct.

At most add a comment about it currently not validating that it's passed an allowed single-character string